### DISCUSSION TITLE
Wgpu backend

### DISCUSSION THREAD

**Discussion by wangjiawen2013** [tracel-ai/burn#2314] [Q&A] [answered]
Hi,
what's the difference between `burn::backend::Wgpu` (https://docs.rs/burn/latest/burn/backend/type.Wgpu.html) and `burn::backend::wgpu::Wgpu` (https://docs.rs/burn/latest/burn/backend/wgpu/type.Wgpu.html) ?

**Comment by laggui** [comment, marked as answer]
It's the same 😅 simply being re-exported [here](https://github.com/tracel-ai/burn/blob/main/crates/burn-core/src/backend.rs#L13L-17)
